tests/commander.configureCommand.test.js

```diff
@@ -84,3 +84,11 @@ test('when storeOptionsAsProperties() after adding option then throw', () => {
     program.storeOptionsAsProperties(false);
   }).toThrow();
 });
+
+test('when storeOptionsAsProperties() after setting option value then throw', () => {
+  const program = new commander.Command();
+  program.setOptionValue('foo', 'bar');
+  expect(() => {
+    program.storeOptionsAsProperties(false);
```

Comment:
Passing `false` is same as the default state, so it isn't the interesting case and pretty unlikely to be encountered:
```suggestion
    program.storeOptionsAsProperties();
```

(I know the throw does not currently care, but I would still like to test the interesting case.)

Reply:
I actually just copied the call from above, and before doing that, I took a minute to think about why you'd decided to pass `false` in that previous test. The way I ended up explaining it to myself is that you'd probably wanted to test the unproblematic call that would not even change anything if it were allowed because that demonstrates how intransigent the check is. It is obvious an error will be thrown when passing `true` or nothing if even passing `false` results in an error (despite the fact that the call would simply be no-op otherwise). That last bit is not so obvious, so that is why I think the `false` case is actually the more interesting one here.

Reply:
> I actually just copied the call from above

Oh, digging... The previous test logic dates back to when the default was to store options as properties, and was not correctly updated when the default behaviour changed. Please change the previous test too! Either pass true or no param, either is fine.